portal/server/configuration-loader.ts

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+type StringBoolean = 'true' | 'false';
+function isStringBoolean(value: string): value is StringBoolean {
+    return value === 'true' || value === 'false';
+}
+function toBoolean(value: string): Boolean {
+    return value === 'true';
+}
+
+
+/**
+* Configuration based on the environment variables.
+*/
+export type Configuration = {
+    edgeConfig?: string;
+    enableBlocklist: Boolean;
+    landingPageOidB36: string;
+    portalDomainNameLength?: number;
+    premiumRpcUrlList: string[];
+    rpcUrlList: string[];
+    enableSentry: Boolean;
+    sentryAuthToken?: string;
+    sentryDsn?: string;
+}
+
+/**
+* A utility class that makes safer to load the environment
```

Comment:
Nit:
```suggestion
* A utility class that makes it safe to load the environment
```
and also line length